src/renderer/zbay/transit.js

```diff
@@ -127,7 +127,7 @@ export const checkMessageSizeAfterComporession = async message => {
 //   return allocatedMessage.toString()
 // }
 
-export const packMemo = async (message, typingIndicato) => {
+export const packMemo = async (message, typingIndicato, onionAddressProp, zcashAddressProp) => {
```

Comment:
In another comment I asked you to remove these new paremeters, but if they were really added here these should be made optional. In any case pleaes remove them.